lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -90,7 +90,7 @@ public Verification withSubject(String subject) {
          */
         @Override
         public Verification withAudience(String... audience) {
-            requireClaim(PublicClaims.AUDIENCE, Arrays.asList(audience));
+            requireClaim(PublicClaims.AUDIENCE, audience);
```

Comment:
Let's use this instead

```suggestion
            requireClaim(PublicClaims.AUDIENCE, audience == null ? null : Arrays.asList(audience));
```